xamarin/Xamarin.Forms
# Wire up Controls.Layouts and Content Page

### Description of Change ###

- wired up Forms.Init so Controls can run successfully
- Wired together various communication points for old style layouts and visual elements
- Moved IFrameWorkElement to VisualElement
- Moved Margin to IView

### Testing Procedure ###
- inside MyApp you'll see some rows you can uncomment to register old renderers
- inside MainPage you'll see a different set of controls you can add that are purely Compatibility Layouts and not new layouts

### PR Checklist ###
